requirements/AUR-CBI-MDL-0100/README.md

```diff
@@ -0,0 +1,11 @@
+# AUR-CBI-MDL-0100 DataStore component shall be compatible with any CBI compliant runtime
+
+It will be possible to deploy DataStore service to any CBI compliant runtime.
+
+# Test
+
+To run the test execute the following command in the terminal:
```

Comment:
Any, for our purpose, means Linux and Leon3. This test would be nice if we could show execution on both.

Reply:
Will be done later